packages/core/src/rules/common/spec-ref-validation.ts

```diff
@@ -0,0 +1,28 @@
+import { Oas2Rule, Oas3Rule } from '../../visitors';
+import { isRef } from '../../ref-utils';
+
+export const SpecRefValidation: Oas3Rule | Oas2Rule = () => {
+  const nodesToValidate = [
+    'Schema',
+    'Response',
+    'Parameter',
+    'RequestBody',
+    'Example',
+    'Header',
+    'SecurityScheme',
+    'Link',
+    'Callback',
+    'PathItem',
+  ];
+
+  return {
+    any(_node, { report, rawNode, rawLocation, type }) {
+      if (!nodesToValidate.includes(type.name) && isRef(rawNode)) {
```

Comment:
It's hard to read :) Could you avoid negation in the statement?

Reply:
I think it's better to have one condition here